src/libp2p_stream_mplex_worker.erl

```diff
@@ -172,7 +180,8 @@ handle_info(Msg, State = #state{mod = Mod}) ->
             {noreply, State}
     end.
 
-handle_packet(Header, Packet, State = #state{mod = Mod}) ->
+handle_packet(Header, Packet, State = #state{mod = Mod, stream_id = StreamId}) ->
+    lager:debug("~p handling packet for stream id ~p, header ~p, packet ~p ~p", [Mod, StreamId, Header, Packet, Packet]),
```

Comment:
Use `_StreamId` like you did in the other clauses where you added debug information?